diff -aur /tmp/D-ITG-2.8.0-rc1/src/ITGSend/ITGSend.cpp ./src/ITGSend/ITGSend.cpp
--- /tmp/D-ITG-2.8.0-rc1/src/ITGSend/ITGSend.cpp	2011-08-07 07:27:50.000000000 -0400
+++ ./src/ITGSend/ITGSend.cpp	2011-10-11 07:39:31.000000000 -0400
@@ -3624,8 +3624,8 @@
 	struct addrinfo * TmpAddress=NULL;	
 	int reuse=1;						
 	int newSock=0;						
-    uint16_t port=0;					
-
+	uint16_t port=0;					
+	timeval timeout={30,0};
 	
 	
 #if defined LINUX_OS && ! defined BSD
@@ -3735,12 +3735,17 @@
 		signalSock = socket(SrcHost->ai_family, SOCK_STREAM, 0);
 
 		if (setsockopt(signalSock, SOL_SOCKET, SO_REUSEADDR, (char*)&reuse, sizeof(reuse))<0){
-			perror("Setsockopt in createTransportChan() (Passive Mode)");
+			perror("Setsockopt reuseaddr in createTransportChan() (Passive Mode)");
 			memClean();
 			exit(1);
 		}
 
-		
+		if (setsockopt(signalSock, SOL_SOCKET, SO_RCVTIMEO, (char*)&timeout, sizeof(timeout))<0){
+			perror("Setsockopt timeout in createTransportChan() (Passive Mode)");
+			memClean();
+			exit(1);
+		}
+	
 		if (bind(signalSock, SrcHost->ai_addr, SrcHost->ai_addrlen) != 0){
 			perror("Bind error in createTransportChan() (Passive Mode)");
 			memClean();
